### ISSUE TITLE
Implementation of  loss function implementation.

### ISSUE THREAD

**Issue by KennyWu** [microsoft/aurora#50]
I am reading the pretraining and finetuning loss function and am trying to understand the implementation of this loss:
![Image](https://github.com/user-attachments/assets/093141d4-ecd1-4076-b3df-46c5523a246c)

I think I have a general idea of implementation, but would like to confirm in case I am not missing any details.
The idea I have is as follows: 

For each variable calculate the MAE loss 
Multiply each variable loss by specific variable weight.
Sum loss of variables grouping by Surface variables and Atmospheric Variables separately
Multiply Surface and Atmospheric losses by alpha and beta weights respectively
Finally sum losses together and multiply by (dataset weight/(tot # variables in dataset)

**Comment by wesselb** [comment]
Hey @KennyWu! I think your interpretation of the equation is correct. :) Also don't forget the division by `H x W` and `C x H x W`, which are important to balance the atmospheric and surface-level variables.

**Comment by firatozdemir** [comment]
Hey @wesselb, I have a follow up question. It looks like the loss is applied on unnormalised predictions, but the dynamic range of different variables are too large to account for the small range you have within w_{k,c}^{A}. 

Can you confirm that the \hat{X}^t and X^t are **not** normalized observations?

**Comment by wesselb** [comment]
Hey @firatozdemir! The loss should be computed over the normalised observations. Otherwise you're completely right that the different magnitudes of the variables would be an issue.
